### ISSUE TITLE
Auto indent on paste doesn't work correctly when pasting on line 1

### ISSUE THREAD

**Issue by stan680** [slynch8/10x#1913]
```C
void function() {
	if (false)
		function();
}
```
gets formatted as
```C
void function() {
	if (false)
	function();
}
```
but only when pasted on line 1

Possibly due to changes regarding #933

**Comment by slynch8** [comment]
I can't get this to happen. You mean like this?

![indentbug](https://github.com/slynch8/10x/assets/7141856/37eda70e-b037-4c26-abcc-821cd70e9073)

**Comment by stan680** [comment]
Ah, I meant when pasting this whole snippet:
![20230717_115802](https://github.com/slynch8/10x/assets/8442114/ec91e817-aabc-488c-9b2c-77151a660b42)

Autoformat selection fixes it.
Looking again now, I'm seeing it happen in other cases than being on line 1 too.

**Comment by slynch8** [comment]
fixed (1.0.58)
